specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json

```diff
@@ -0,0 +1,4685 @@
+              }
+            ]
+          }
+        },
+        "country": {
+          "description": "ISO country code. Country for hardware shipment. For codes check: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements",
+          "type": "string"
+        },
+        "location": {
+          "description": "Location for data transfer. For locations check: https://management.azure.com/subscriptions/SUBSCRIPTIONID/locations?api-version=2018-01-01",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "ValidateSkuAvailability"
+    },
+    "SkuAvailabilityValidationResponseProperties": {
+      "description": "Properties of sku availability validation response.",
+      "required": [
+        "validationType"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputResponse"
+        }
+      ],
+      "properties": {
+        "status": {
+          "description": "Sku availability validation status.",
+          "enum": [
+            "Valid",
+            "Invalid",
+            "Skipped"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ValidationStatus",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "Valid",
+                "description": "Validation is successful"
+              },
+              {
+                "value": "Invalid",
+                "description": "Validation is not successful"
+              },
+              {
+                "value": "Skipped",
+                "description": "Validation is skipped"
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-discriminator-value": "ValidateSkuAvailability"
+    },
+    "SkuCapacity": {
+      "description": "Capacity of the sku.",
+      "type": "object",
+      "properties": {
+        "usable": {
+          "description": "Usable capacity in TB.",
+          "type": "string",
+          "readOnly": true
+        },
+        "maximum": {
+          "description": "Maximum capacity in TB.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SkuCost": {
+      "description": "Describes metadata for retrieving price info.",
+      "type": "object",
+      "properties": {
+        "meterId": {
+          "description": "Meter id of the Sku.",
+          "type": "string",
+          "readOnly": true
+        },
+        "meterType": {
+          "description": "The type of the meter.",
+          "type": "string",
+          "readOnly": true
+        },
+        "multiplier": {
+          "format": "double",
+          "description": "Multiplier specifies the region specific value to be multiplied with 1$ guid. Eg: Our new regions will be using 1$ shipping guid with appropriate multiplier specific to region.",
```

Comment:
users know what this means?

Reply:
> users know what this means?

Yes the main consumer of this is databox portal team.